src/helpers/table-sort-helper.js

```diff
@@ -19,7 +38,8 @@ export function stableSort(array, cmp) {
 }
 
 export function getSorting(order, orderBy) {
+  const descFunction = typeof orderBy === "number" ? statusDesc : desc;
```

Comment:
I don't believe this is sustainable, but it will do until we need to sort on something else.